__tests__/app.test.js

```diff
@@ -0,0 +1,72 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => db.end());
+beforeEach(() => seed(data));
+
+describe("GET /api/topics", () => {
+  test("Get:200 return  all topics ",() => {
+      return request(app).get("/api/topics").expect(200);
+  });
+    
+    test("responds with topics data in array", () => {
+      return request(app)
+        .get("/api/topics")
+        .then(({ body }) => {
+            const { topic } = body;
+          expect(topic).toBeInstanceOf(Array);
+        });
+    });
+
+
+
+      test(" return a length of topic array", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
+          .then(({ body }) => {
+            const { topic } = body;
+            expect(topic.length).toBe(3);
+          });
+      });
+    
+    test("Return objects should have slug and description properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+          .then(({ body }) => {
+            const {topic} = body
+          topic.forEach((topic) => {
```

Comment:
Your previous test where you check the array should ideally be in this test, just before the .forEach :)